kalite/distributed/urls.py

```diff
@@ -38,6 +38,10 @@
     url(r'^securesync/', include(securesync.urls)),
 )
 
+
+# TODO: This should only be in DEBUG settings and the HTTP server should be
+# serving it otherwise. But the cherrypy service does not currently serve
+# media by itself, it uses the django application handlers below.
```

Comment:
Hmm, not sure why this would be needed outside of `DEBUG = True`. Cherrypy *does* serve media by itself:
https://github.com/learningequality/ka-lite/blob/develop/kalite/django_cherrypy_wsgiserver/cherrypyserver.py#L49